src/pages/_app.tsx

```diff
@@ -29,6 +30,7 @@ const App = ({ Component, pageProps, ...props }: AppPropsWithProps) => {
 App.getInitialProps = async () => {
   return {
     props: {
+      exImmichUrl: ENV.EXTERNAL_IMMICH_URL,
```

Comment:
Actually we wont use `immichURL` if its internal in the frontend, we can directly replace this key with `ENV.EXTERNAL_IMMICH_URL`

Reply:
yeah thanks will be fixed.

Reply:
Hey i think we should keep immichurl as we are using it in the connection info section
```
 <p className="text-muted-foreground font-mono">Connected to (Internal)</p>
        <Link href={immichURL} target="_blank" className="font-mono">
          {immichURL}
        </Link>
```